rbac/management/tenant_service/v2.py

```diff
@@ -262,6 +269,27 @@ def _bootstrap_tenant(self, tenant: Tenant) -> BootstrappedTenant:
 
         return BootstrappedTenant(tenant, mapping, default_workspace=default_workspace, root_workspace=root_workspace)
 
+    def _replicate_bootstrap(self, tenant: Tenant, mapping: TenantMapping):
+        """Replicate the bootstrapping of a tenant."""
+        built_in_workspaces = Workspace.objects.filter(
+            tenant=tenant, type__in=[Workspace.Types.ROOT, Workspace.Types.DEFAULT]
```

Comment:
Definitely not in scope for this, but it might be nice at some point if we had a custom model manager that could return the built-ins with something like `Workspace.objects.built_ins(tenant=tenant)`. A future DRY task, maybe. 😄